atomic_reactor/plugin.py

```diff
@@ -408,8 +408,8 @@ def __init__(self, dt, workflow, plugin_conf, *args, **kwargs):
         logger.info("initializing runner of build-step plugin")
         self.plugins_results = workflow.buildstep_result
 
-        if not plugin_conf:
-            # if none buildstep plugins specified, fallback to docker api plugin
+        if plugin_conf is None:
```

Comment:
Just that if the build JSON explicitly says not to run any buildstep plugins, we shouldn't (and this will fail).

It's minor: in practice it would only happen if there was an osbs-client misconfiguration or something.